admin/templates/configuration/etc/log4cxx.unittest.properties

```diff
@@ -0,0 +1,16 @@
+#
```

Comment:
I do not think this is the right place for log config file. Unit tests are running when we build stuff at which point run directory does not even exist. Another issue with that is that the number of standard log4cxx config files is supposed to be small, preferably one, instead we area adding more and more files. Can we survive without this particular file? Why do unit tests need special log4cxx config?